docs/keycloak-configuration.md

```diff
@@ -26,153 +33,313 @@ To create a new realm follow the steps below:
 - Keep "Client authentication" OFF - as the OIDC type is not confidential access. When set to OFF it is a public access type that we want for our front-end application, as it is a JavaScript-based browser application
 - Keep "Authorization" to OFF - as we want fine-grained authorization support disabled for our client
 - Under "Authentication flow" enable "Standard flow" and disable everything else
+
+Standard flow enables standard OpenID Connect redirect-based authentication with an authorization code. In terms of OpenID Connect or OAuth2 specifications, this enables support of 'Authorization Code Flow' for this client.
+
+Authorization flow with public access is chosen to support redirects after login and logout sessions.
+
+With the authorization flow, the user is sent to the Keycloak login page when accessing the application and is allowed to log in with credentials or register for a new account.
+
 - Click Save
 
+![Manually Create Client - Client Capability Config Page](/docs/rsrc/keycloak_diagrams/17-client-capability-config.png)
+
 ## Setting Client Access Settings
 
 Stay on the newly created client "Clients" > "Tornjak-React-auth" panel, in the "Settings" tab:
 
+![Client Settings Page](/docs/rsrc/keycloak_diagrams/18-client-settings-page.png)
+
 Under the client Access settings sub-section:
-Set your "Valid redirect URIs" - for example, http://localhost:3000/* - this is an access point to the Tornjak application. If the application is run locally such as on (minikube or kind) this will be localhost, but if the Tornjak runs on the server, it would be an ingress value of the service. [This is the URI you want Keycloak to redirect to after successful login]
-Set your "Valid post logout redirect URIs" - for example, http://localhost:3000/* [This is the URI you want Keycloak to redirect to after successful logout]
-Set "Web origins" as '*' (Wildcard to allow all origins)
+- Set your "Valid redirect URIs" - for example, http://localhost:3000/* - this is an access point to the Tornjak application. If the application is run locally such as on (minikube or kind) this will be localhost, but if the Tornjak runs on the server, it would be an ingress value of the service. [This is the URI you want Keycloak to redirect to after successful login]
+- Set your "Valid post logout redirect URIs" - for example, http://localhost:3000/* [This is the URI you want Keycloak to redirect to after successful logout]
+- Set "Web origins" as '*' (Wildcard to allow all origins)
 
 This is the allowed CORS origins. You can explicitly assign such as where your application running - for example, http://localhost:3000
 To permit all origins of Valid Redirect URIs, explicitly add '*'.
-Click on Save
 
-Disabling Full scope Enabled Feature
+- Click on Save
+
+![Client Access Settings Page](/docs/rsrc/keycloak_diagrams/19-client-access-settings.png)
+
+## Disabling Full scope Enabled Feature
```

Comment:
from what i understand from personally doing the partial import, everything else is configured but the full scope is still enabled. to make sure of the clear distinction maybe bring this to the top with the"import realm" in section 1